.github/workflows/dependency-updates-post-processing.yaml

```
@@ -45,17 +30,22 @@ jobs:
           submodules: true
           token: ${{ secrets.APP_AUTOSCALER_CI_TOKEN }} # With push token that can trigger new PR jobs
       - name: Configure git
+        id: configure_git
         shell: bash
         run: |
           #! /usr/bin/env bash
           set -eu -o pipefail
 
-          declare -r commit_author_name="${{github.event.head_commit.author.name}}"
-          declare -r commit_author_email="${{github.event.head_commit.author.email}}"
+          declare -r commit_author_name="$(git log -1 --pretty=format:'%an')"
```

personal preference, totally up to you

```suggestion
          readonly commit_author_name="$(git log -1 --pretty=format:'%an')"
```

---

Interesting!


https://stackoverflow.com/a/56628154 presents an argument for using `local -r` and `readonly`, but on the other hand `declare` feels like the more general approach.

---

totally up to you! : )